wadas/ui/configure_web_interface.py

```diff
@@ -133,16 +170,33 @@ def update_web_interface_status(self):
     def on_web_interface_start_clicked(self):
         """Method to trigger start of web interface"""
 
-        #TODO: add logic to start web interface start
-        self.web_interface_enabled = True
-        self.update_web_interface_status()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+        webserver_dir = os.path.join(current_dir, "../../wadas_webserver")
+        script_path = os.path.join(webserver_dir, self.web_interface_main_file)
```

Comment:
Using pathlib.Path simplifies this path handling, and it also allows to avoid using the os.* functions (below).